modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -77,41 +77,51 @@ definition {
 	@description = "LPS-151877 - Verify that, when adding a new Object, the admin user can activate Account Restrictions if there is a relationship between the created Object and the System Object Account"
 	@priority = "4"
 	test CanActivateAccountRestriction {
-		ObjectAdmin.addObjectViaAPI(
-			labelName = "Custom Object 156819",
-			objectName = "CustomObject156819",
-			pluralLabelName = "Custom Objects 156819");
+		task ("Add Custom Object (via API)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 156819",
+				objectName = "CustomObject156819",
+				pluralLabelName = "Custom Objects 156819");
+		}
```

Comment:
We should be making these changes in the commit that added the code. Next time, please include the comments in the original commit instead of making a new commit.